src/prototypes/draft-email-textarea/js/draft-email-textarea.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * The contents of this function will be inlined in a Gutenberg Custom HTML block
+ */
+export const runProposedInlineJS = () => {
+  // The elements all will be a part of the same Gutenberg Custom HTML block.
+  // I chose to use IDs since the HTML and JS are scoped to the same Gutenberg
```

Comment:
Total nit: if we're building this as a block it's _possible_ for the block to be included multiple times on one page. If that were the case then we'd have issues with duplicate IDs. That said, I think that's a _really_ unlikely use case, so I'm fine ignoring it here.

Reply:
Thanks, @Paul-Hebert. Valid concern.

As I understand, a Custom HTML block cannot be included multiple times. I'm not creating a "custom" reusable block. Instead, I'm only inlining HTML/CSS/JS in the Custom HTML block. The only way that there might be duplicates is if I copy-paste whatever is inside the Custom HTML block into another Custom HTML block.

This use case is a one-off for now, so I don't see a need to copy-paste into various Custom HTML blocks.